src/utilities/sql/SqlFile.cpp

```diff
@@ -1733,4 +1733,12 @@ bool SqlFile::hasYear() const {
   return true;
 }
 
+bool SqlFile::hasIlluminanceMapYear() const {
```

Comment:
New method since `hasYear` works for 8.9.0 (or 9.0.1 at least) but the illuminance map added year only in 9.2.0